tests/conftest.py

```
@@ -0,0 +1,183 @@
+            text=True,
+            env=env
+        )
+
+        # Wait for start
+        started = False
+        for _ in range(40): # 20 seconds
+            try:
+                requests.get(f"{DEMO_URL}/api/health", timeout=0.5)
+                started = True
+                break
+            except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)